lms/templates/courseware/courseware.html

```diff
@@ -126,10 +126,10 @@
                 <div class="search-field-wrapper">
                   <input id="course-search-input" type="text" class="search-field"/>
                   <button type="submit" class="search-button">
-                    ${_('search')} <i class="icon fa fa-search" aria-hidden="true"></i>
+                    ${_('search')} <span class="icon fa fa-search" aria-hidden="true"></span>
                   </button>
                   <button type="button" class="cancel-button" aria-label="${_('Clear search')}">
```

Comment:
Do you mean switch 'Clear search' to 'Cancel search' because of the cancel button? I thought clear search might have been a better descriptor than cancel search because when I click on the x in a textbox, I view it as clearing the textbox, not cancelling a potential search.